.circleci/config.yml

```diff
@@ -3,6 +3,7 @@ version: 2.1
 orbs:
   codecov: codecov/codecov@3
   win: circleci/windows@5.0.0
+  macos: circleci/macos@2.4.1
```

Comment:
It's probably fine to reduce precision to minor or major version here, that way the latest is used.